src/pages/[boardId]/create-paper/MessageStep.tsx

```
@@ -0,0 +1,69 @@
+/** @jsxImportSource @emotion/react */
+import {Button} from '@/components/Button';
+import FixedBottomCTA from '@/components/FixedBottomCTA/FixedBottomCTA';
+import {VStack} from '@/components/Stack';
+import {TextArea} from '@/components/TextArea/TextArea';
+import Top from '@/components/Top/Top';
+import SETTING from '@/constants/setting';
+import {useRequestPostPaper} from '@/hooks/useRequestPostPaper';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)